File: mochi/src/main/java/com/vitorsanches/mochi/modules/anime/Season.java
package com.vitorsanches.mochi.modules.anime;

import lombok.Data;

public enum Season {
    FALL("fall"),
    WINTER("winter"),
    SPRING("spring"),
    SUMMER("summer");

    private final String value;

    Season(String value) {
        this.value = value;
    }

    public String getValue() {
        return this.value;
    }

    @Override
    public String toString() {
        return value;
    }
}

File: mochi-ui/src/common/i18n/season.ts
import Season from '../data-types/season.enum';

export const seasonPtBrMap = new Map([
  [Season.FALL, 'Outono'],
  [Season.WINTER, 'Inverno'],
  [Season.SPRING, 'Primavera'],
  [Season.SUMMER, 'Verão']
]);

export const seasonEnMap = new Map([
  [Season.FALL, 'Fall'],
  [Season.WINTER, 'Winter'],
  [Season.SPRING, 'Spring'],
  [Season.SUMMER, 'Summer']
]);

export const seasonEsMap = new Map([
  [Season.FALL, 'Caer'],
  [Season.WINTER, 'Invierno'],
  [Season.SPRING, 'Primavera'],
  [Season.SUMMER, 'Verano']
]);

export const seasonMapByLang = {
  'pt-br': seasonPtBrMap,
  'es': seasonEsMap,
  'en': seasonEnMap,
}
